airflow/www/extensions/init_react_ui.py

```diff
@@ -33,6 +33,7 @@ def init_react_ui(app):
     )
 
     @bp.route("/ui")
+    @bp.route("/ui/dags")
```

Comment:
I really don't like this. I am very open to suggestions on how to make Flask AppBuilder ignore all `/ui` endpoints without having to specify each one in python.

Reply:
Yes that's not great, and the coupling with the UI router :( 

Maybe we can get away with custom converters https://werkzeug.palletsprojects.com/en/3.0.x/routing/#custom-converters. (Something a regexp that catches `/ui*` basically)

Reply:
Oh I got it working for one layer of paths but we will still need to add a custom converter for more nested paths.